### DISCUSSION TITLE
How i can change default header token or remove it from request ?

### DISCUSSION THREAD

**Discussion by tungthanhss** [ng-alain/ng-alain#2444] [General]
![image](https://github.com/ng-alain/ng-alain/assets/34281514/c5196171-db8a-4fed-a9a5-f18480d15a45)

**Comment by tungthanhss** [comment]
I try to set place to body, but seem like it's temporarily
    token_send_place: 'body',
    token_send_template: 'Bearer ${token}',
    token_send_key: 'Authorization',

**Reply by cipchk** [reply]
Try use `ALLOW_ANONYMOUS`:

```ts
this.http.get(`/path`, { context: new HttpContext().set(ALLOW_ANONYMOUS, true) })
```
